src/drafts/MarkdownEditor/_MarkdownInput.tsx

```diff
@@ -118,7 +118,7 @@ export const MarkdownInput = forwardRef<HTMLTextAreaElement, MarkdownInputProps>
             width: '100%',
             borderStyle: 'none',
             height: fullHeight ? '100%' : undefined,
-            boxShadow: isDraggedOver ? 'primer.shadow.focus' : 'none',
+            outline: isDraggedOver ? 'solid 2px color.accent.fg' : undefined,
```

Comment:
Is this valid markup? I didn't see any stories for this code ☹️ should I add one?